modules/apps/object/object-test/src/testFunctional/tests/Picklist.testcase

```diff
@@ -531,12 +531,80 @@ definition {
 	}
 
 	@description = "LPS-136595 - Verify that the translated Picklist item's name will be displayed on Forms"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test ViewTranslatedPicklistItemNameOnForms {
+		Picklist.addPicklistViaAPI(picklistName = "Picklist Test");
+
+		Picklist.addPicklistItemViaAPI(
+			itemKey = "PicklistItemKeyTest",
+			itemName = "Picklist Item Name Test",
+			picklistName = "Picklist Test");
+
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 147730",
+			objectName = "CustomObject147730",
+			pluralLabelName = "Custom Objects 147730");
+
+		Picklist.gotoPicklists();
+
+		Picklist.optionsPicklist(
+			picklistName = "Picklist Test",
+			picklistOption = "View");
+
+		Picklist.addLanguageOnPicklist(
+			itemName = "Teste na lista",
+			newlanguage = "pt-BR");
+
+		Picklist.optionsPicklistItem(
+			itemKey = "PicklistItemKeyTest",
+			picklistItemOption = "View");
+
+		Picklist.addLanguageOnPicklistItem(
+			itemName = "Teste de entrada",
+			newlanguage = "pt-BR");
+
+		Button.clickSave();
+
+		Navigator.openURL();
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object 147730");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Field Picklist",
+			fieldPicklist = "Picklist Test",
+			fieldType = "Picklist");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject147730");
+
+		User.openUsersAdmin();
+
+		User.editDisplaySettingsCP(
+			languageName = "português (Brasil)",
+			userScreenName = "test");
+
+		Navigator.openURL();
+
+		FormsAdmin.addForm();
+
+		PopulateObjectFormsConfiguration.openFormSettings();
+
+		PopulateObjectFormsConfiguration.selectObject(label = "Objeto");
+
+		PopulateObjectFormsConfiguration.selectCustomObject(label = "Custom Object 147730");
 
-		// TODO LPS-147730 ViewTranslatedPicklistItemNameOnForms pending implementation
+		Form.gotoAddField(
+			fieldType = "Selecionar da lista",
+			localeFields = "Campos");
 
+		Form.gotoAdvancedTab();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
 	}
```

Comment:
Hi, @vsigor!

This test is not part of the commit, it has been blocked. I updated your test in this PR: https://github.com/rodrigocunhaa/liferay-portal/pull/1790

I will go closed this PR. Thanks!